util/__init__.py

```diff
@@ -580,9 +580,9 @@ def iso_639_2_for_locale(cls, locale):
             language, place = locale.lower().split("-",1)
         else:
             language = locale
-        if language in cls.two_to_three:
+        if cls.two_to_three[language]:
```

Comment:
I was encountering a problem where `'eng' in cls.two_to_three` was True even though `cls.two_to_three['eng']` was None. I haven't been able to reproduce this problem but this change is a more robust way to express the same underlying idea as the old code.